stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -409,7 +434,13 @@ fn test_simple_nakamoto_coordinator_10_tenures_10_blocks() {
                         &recipient_addr,
                     );
 
-                    vec![stx_transfer]
+                    let aggregate_tx = make_pox_4_aggregate_key(
```

Comment:
Why do we need to set the aggregate public key multiple times (e.g. in each block)?

Reply:
Oh I see, we still need to add a way for the node itself to do this as part of processing a reward cycle.

Reply:
It probably does not need to be EVERY block. I just decided lets do overkill to avoid missing processing it during replay. In theory this should not be needed as we simulate signer's submitting their votes across a reward cycle rather than a straight "set-aggregate-public-key" call. (this call should be removed once we have the ability to vote correctly)